tools/exploit/nasm_shell.rb

```diff
@@ -17,8 +17,6 @@
 end
 
 $:.unshift(File.expand_path(File.join(File.dirname(msfbase), '..', '..', 'lib')))
-require 'msfenv'
```

Comment:
This breaks the tool:

```
wvu@kharak:/rapid7/metasploit-framework:master$ tools/exploit/nasm_shell.rb
Traceback (most recent call last):
	2: from tools/exploit/nasm_shell.rb:42:in `<main>'
	1: from tools/exploit/nasm_shell.rb:42:in `new'
/rapid7/metasploit-framework/lib/rex/ui/text/output/stdio.rb:36:in `initialize': undefined method `assert_valid_keys' for {}:Hash (NoMethodError)
wvu@kharak:/rapid7/metasploit-framework:master$
```

Was this tested?

Reply:
yes, sorry, forgot to restore msfenv 😅

Reply:
Seems like this `Rex::Ui::Text::Output::Stdio` is depending on `assert_valid_keys` which actually comes from activesupport. This seems like something that could be fixed without much effort, but happy to leave this to a separate PR :+1: